cloud: add release note for 20230110 (#11971)
pingcap/docs: tidb-cloud/tidb-cloud-release-notes.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.

## January 10, 2023

**General changes**

> - Optimize the feature of importing data from local CSV files to TiDB to improve user experience.

Author: ```suggestion
- Optimize the feature of importing data from local CSV files to TiDB to improve user experience for [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
```

> - Optimize the feature of importing data from local CSV files to TiDB to improve user experience for [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.

Author: ```suggestion
- Optimize the feature of importing data from local CSV files to TiDB to improve the user experience for [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
```

    - To upload a CSV file, now you can simply drag and drop it to the upload area on the **Import** page.

>     - When configuring an import task, if your target database or table does not exist, you can enter a name to let TiDB Cloud create it for you automatically. For the new target table, you can specify the primary key and select multiple fields to form a composite primary key.

Reviewer: ```suggestion
    - When creating an import task, if your target database or table does not exist, you can enter a name to let TiDB Cloud create it for you automatically. For the newly created table, you can specify a primary key or select multiple fields to form a composite primary key.
```

    - After the import is completed, you can explore your data with [AI-powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md) by clicking **Explore your data by Chat2Query** or clicking the target table name in the task list.

>     For more information, see [Import Local Files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).

Author: ```suggestion
  For more information, see [Import Local Files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).
```
